aws-native-python/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pulumi>=3.0.0,<4.0.0
+pulumi-aws-native>=0.8.0,<2.0.0
```

Comment:
Yeah, was debating this with myself. Assume we'd want them to pick up version 1 when it comes out, but probably not the following major version change. Think I'll go with <1 in case there's breaking changes when we go the first stable release.

Reply:
<1 makes sense as it's comparable with what we set for all other languages. Also, pulumi 4.0 is likely to come before aws-native 1.0 anyway.